src/pages/search_page/search_page.js

```diff
@@ -50,26 +62,89 @@ export default function SearchPage() {
   useEffect(() => {
     setUrlParams({ ...urlParams, ...mapCoordinates }, history);
     // eslint-disable-next-line
-  }, [mapCoordinates, history]);
+  }, [mapCoordinates, history,]);
```

Comment:
:thinking: something wrong, i can feel it. Why are you ignore urlParams in dependency list?